test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -4074,7 +4074,7 @@ items:
   metadata:
     annotations:
       authorization.openshift.io/system-only: "true"
-      rbac.authorization.kubernetes.io/autoupdate: "true"
+      openshift.io/reconcile-protect: "false"
```

Comment:
I think we need to update our bootstrap policy to always have these annotations (they only exist for roles we get from RBAC and convert into origin).

@deads2k @liggitt which annotation would you expect to be used in this file?